rest-api-spec/src/main/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -163,12 +163,16 @@ be caught and tested.  For instance:
 The argument to `catch` can be any of:
 
 [horizontal]
-`missing`::     a 404 response from ES
-`conflict`::    a 409 response from ES
-`request`::     a generic error response from ES
-`param`::       a client-side error indicating an unknown parameter has been passed
-                to the method
-`/foo bar/`::   the text of the error message matches this regular expression
+`unauthorized`::    a 401 response from ES
+`forbidden`::       a 403 response from ES
+`missing`::         a 404 response from ES
+`request_timeout`:: a 408 response from ES
+`conflict`::        a 409 response from ES
+`unavailable`::     a 503 response from ES
+`request`::         a generic error response from ES
```

Comment:
I wonder if this should be clarified to say that it does not include 401, 403, 404, 408, and 409 and is otherwise any 400 or 500?